packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -250,14 +261,20 @@ export class GalleryContainer extends React.Component {
         styleParams: styles,
         allowPreloading,
       });
-      this.videoScrollHelper.updateGalleryStructure({
+
+      const scrollHelperNewGalleryStructure = {
```

Comment:
again - the helper should handle its state.
It knows whether it was inited or not